### PR TITLE
Docs: concept information and new Console screenshot

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/ref-resources-table-service-admin.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+The following table describes the {ProductName} resources that pertain to the service administrator role.
+
+.{ProductName} service administrator resources table
+[cols="50%a,50%a",options="header",subs="attributes"]
+|===
+|Resource |Description
+|`addressplans`|
+|`addressspaceplans`|
+|`brokeredinfraconfigs`|Specifies the infrastructure configuration for brokered address spaces?
+|`standardinfraconfigs`|Specifies the infrastructure configuration for standard address spaces?
```
Comment (lulf) [inline documentation/modules/ref-resources-table-service-admin.adoc]: This seems incomplete

File: documentation/modules/ref-resources-table-messaging-tenant.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The following table describes the {ProductName} resources that pertain to the messaging tenant role.
+
+.{ProductName} messaging tenant resources table
+[cols="50%a,50%a",options="header",subs="attributes"]
+|===
+|Resource |Description
+|`addresses`|Specifies the address.
+|`addressspaces`|Specifies the address space.
+|`addressspaceschemas`|Specifies .
```
Comment (jenmalloy) [inline documentation/modules/ref-resources-table-messaging-tenant.adoc]: @k-wall, Can you please help with a proper description of `addressspaceschemas`?

File: documentation/modules/con-console.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+//
+// assembly-using-console.adoc
+
+[id='con-console-{context}']
+= {ConsoleName} user permissions
+
+{ConsoleName} uses the OpenShift RBAC permissions model. For more information about the OpenShift RBAC permissions model, see the link:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html[OpenShift documentation^].
+
+To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
+
```
Comment (jenmalloy) [inline documentation/modules/con-console.adoc]: @k-wall, Can you please review this file for accuracy?

File: documentation/modules/con-console.adoc [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-console.adoc
+
+[id='con-console-{context}']
+= {ConsoleName} user permissions
+
+{ConsoleName} uses the OpenShift RBAC permissions model. For more information about the OpenShift RBAC permissions model, see the link:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html[OpenShift documentation^].
+
+To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
```
Comment (k-wall) [inline documentation/modules/con-console.adoc]: Terminology: Addressspaces and addresses are **resources**.  A **role** object defines a set of permissions that apply to one or more resources.

Comment (k-wall) [inline documentation/modules/con-console.adoc]: "the OpenShift user requires `addressspace` and `address` "  .. In reality the user requires addresspace permission to reach the first page in the console (that lists the addressspaces_, then they need address permission to click through see the addresses, but I think what you have said is good enough detail for here.

File: documentation/modules/ref-resources-table-messaging-tenant.adoc (hunk 2 above)
Comment (k-wall) [reply documentation/modules/ref-resources-table-messaging-tenant.adoc]: an `addressspaceschema` defines the service characteristics available to an addresspace.  An addressspace refers one addressspaceschema.  'standard' and 'brokered' are predefined addressspaceschemas.

File: documentation/modules/con-console.adoc (hunk 4 above)
Comment (jenmalloy) [reply documentation/modules/con-console.adoc]: Fixed

Comment (jenmalloy) [reply documentation/modules/con-console.adoc]: Ok, thanks for the clarification. I left it as is.

File: documentation/modules/ref-resources-table-messaging-tenant.adoc (hunk 2 above)
Comment (jenmalloy) [reply documentation/modules/ref-resources-table-messaging-tenant.adoc]: Fixed both issues

File: documentation/modules/con-console.adoc [hunk 5]
```diff
@@ -7,6 +7,6 @@
...
 
-To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
+To access the {ConsoleName}, the OpenShift user requires the `addressspace` and `address` resources to be included in the role object, which defines a set of permissions that apply to one or more resources. The specific permissions given to the role object govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the associated role object, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
```
Comment (k-wall) [inline documentation/modules/con-console.adoc]: Mmm.  I'm not sure about the first sentence.  It is not wrong but overly detailed.  We are already supplying a link to 3.11/admin_guide/manage_rbac.html so the user can easily get the detail from the authoritative source.

How about:

"To use the {ConsoleName}, the OpenShift user requires a role that grants access to `addressspace` and `address` resources."

File: documentation/modules/con-console.adoc (hunk 4 above)
Comment (k-wall) [reply documentation/modules/con-console.adoc]: I left one new suggestion.

**Pull request merged**